templates/dbstats-cronjob.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.dbstats.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "common.fullname" ( dict "root" . "service" .Values.dbstats ) }}
+  labels: {{ include "common.labels" ( dict "root" . "service" .Values.dbstats ) | nindent 4 }}
+  annotations: {{- toYaml .Values.dbstats.annotations | nindent 4 }}
+spec:
+  schedule: "{{ mod (regexReplaceAll "[a-f]+" (printf "%s-%s" .Release.Namespace .Release.Name | sha256sum) "" | trunc 5) 60 }} * * * *"
```

Comment:
this is just to add a bit of randomnes to the scheduled time to avoid that all the metrics are send at the same time concurrently isn't it?